sci-biology/bru2nii/bru2nii-9999.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+
+inherit git-r3
+
+DESCRIPTION="Converts Bruker ParaVision MRI data to the NIfTI file format"
+HOMEPAGE="https://github.com/neurolabusc/Bru2Nii"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/neurolabusc/Bru2Nii.git"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+IUSE="gui"
+
+RDEPEND=""
+DEPEND="dev-lang/fpc
+	gui? ( dev-lang/lazarus )"
+
+src_compile() {
+	fpc -Tlinux Bru2.lpr || die
+	use gui && (lazbuild -B --lazarusdir="/usr/share/lazarus/" Bru2Nii.lpr || die)
```

Comment:
Please check https://devmanual.gentoo.org/ebuild-writing/error-handling/index.html#die-and-subshells

You need to use an if then clause here.